source/systems/gaea_user_guide.rst

```diff
@@ -91,2113 +93,1545 @@ points at Atlanta and Chicago.
 
     * IBM Spectrum Scale
 
-
     * 75 PB
 
     * IBM Elastic Storage Server 3500 running GPFS 5.1
 
-
-
 Gaea is the largest of the four NOAA RDHPCS, and is used to study the
 earth's notoriously complex climate from a variety of angles by
 enabling scientists to:
 
 * understand the relationship between
 
   * climate change and extreme weather such as hurricanes
-  * the atmosphere’s chemical makeup and climate
+  * the atmosphere's chemical makeup and climate
 
```

Comment:
Recommend you eliminate the sub-bullets, they make things less clear. Suggest:

- to understand the relationship between climate change and extreme weather, and the atmosphere's chemical makeup and climate
- to investigate the climate role played by the oceans that cover nearly three-quarters of the globe.

Reply:
Adjusted in a new push.